CCDB/src/CcdbApi.cxx

```diff
@@ -161,7 +161,9 @@ void CcdbApi::init(std::string const& host)
     }
     snapshotReport = fmt::format("(cache snapshots to dir={}", mSnapshotCachePath);
   }
-  if (getenv("IGNORE_VALIDITYCHECK_OF_CCDB_LOCALCACHE")) {
+
+  const char* ignoreValidity = getenv("IGNORE_VALIDITYCHECK_OF_CCDB_LOCALCACHE");
+  if ((cachedir && !ignoreValidity) || (ignoreValidity && atoi(ignoreValidity) == 1)) {
```

Comment:
OK, but then, should not it be 
```
if ((cachedir && !ignoreValidity) || (ignoreValidity && atoi(ignoreValidity) != 0)) {
```
?

Fine for me to ignore by default the timestamps of the cached object. But my point is that if we decide to not ignore them, then it does not work with the current code. So, we either give up this option completly (which is fine for me) or fix it.